GitCommands/SshPathLocator.cs

```diff
@@ -68,5 +74,13 @@ public string Find(string gitBinDirectory)
                 return null;
             }
         }
+
+        /// <summary>
+        /// Check if a path is a known OpenSSH executable
```

Comment:
The implementation is quite different from what the doc states.
```suggestion
        /// Checks if OpenSSH is accessible via %PATH%, or comes from the git's bin folder.
```

Reply:
Will remove this method, add an out parameter to Find() instead to inform  if this is an OpenSSH - or we go the full way and set it to empty if matching OpenSSH in the Git bindir. (The last alternative may be OK, I was not sure when I did this initially.)

Reply:
That seems like a better solution.